pcl_segmentation occur compile error on MSVC2010

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Operating System and version: Windows7 32bit
* Compiler: Visual Studio 2010
* PCL Version: PCL 1.8.0

When I compile the newest pcl-master with Visual Studio 2010, pcl_segmentation seems to occur some errors:

```
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	179
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	179  
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	180  
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	180  
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	181  
error C2487: pcl::octree::OctreePointCloud<PointT,LeafContainerT,BranchContainerT>::addPointToCloud”: member of dll interface class may not be declared with dll interface	D:\Github\PCL\source\segmentation\src\supervoxel_clustering.cpp	181  
```

The codes with errors are below:
```
template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZ, AdjacencyContainerT>;
template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGB, AdjacencyContainerRGBT>;
template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGBA, AdjacencyContainerRGBAT>;
```

No errors occur if I just annotate these 3 lines. But I wonder whether it has any negative effect.

--- a/segmentation/src/supervoxel_clustering.cpp
+++ b/segmentation/src/supervoxel_clustering.cpp
@@ -168,14 +168,14 @@ typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZ, VoxelData
 typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGB, VoxelDataRGBT> AdjacencyContainerRGBT;
 typedef pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGBA, VoxelDataRGBAT> AdjacencyContainerRGBAT;
 
-template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZ>;
-template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZRGB>;
-template class PCL_EXPORTS pcl::SupervoxelClustering<pcl::PointXYZRGBA>;
+template class pcl::SupervoxelClustering<pcl::PointXYZ>;
+template class pcl::SupervoxelClustering<pcl::PointXYZRGB>;
+template class pcl::SupervoxelClustering<pcl::PointXYZRGBA>;
 
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZ, VoxelDataT>;
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGB, VoxelDataRGBT>;
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGBA, VoxelDataRGBAT>;
+template class pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZ, VoxelDataT>;
+template class pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGB, VoxelDataRGBT>;
+template class pcl::octree::OctreePointCloudAdjacencyContainer<pcl::PointXYZRGBA, VoxelDataRGBAT>;
 
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZ, AdjacencyContainerT>;
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGB, AdjacencyContainerRGBT>;
-template class PCL_EXPORTS pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGBA, AdjacencyContainerRGBAT>;
+template class pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZ, AdjacencyContainerT>;
+template class pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGB, AdjacencyContainerRGBT>;
+template class pcl::octree::OctreePointCloudAdjacency<pcl::PointXYZRGBA, AdjacencyContainerRGBAT>;

